text/0025-purchase-order.md

```diff
@@ -309,31 +308,30 @@ Beginning state. An organization with po::draft permissions created a version
 for collaboration purposes.
 
 Editable state. The contents of the version are being populated by one or more
-organizations. Any organization has permission to submit the draft for review
-or close it in favor of a better version.
+organizations. Any organization with po::draft permissions can submit the draft for review
+or close it in favor of a different version.
 
 Review state. An organization has proposed the version for review by another
 organization. Another organization has permission to approve the version by
-moving it to Composed state. The organization may also decline the version by
-adding a rejection reason (optional) or move it back to Editable state for
+moving it to Composed state. The organization may also decline the version,
+with the option to add a rejection reason, or move it back to Editable state for
 further modification.
 
-Composed state. Both organizations are in agreement on the contents of the
-version. The buying organization retains the decision to issue a formal
+Composed state. Both organizations have accepted the contents of the
+version. The buying organization may decide to issue a formal
 purchase order based on the drafted contents.
 
-Declined state. An organization has declined the version content. An
+Declined state. An organization has declined the version content. Any
 organization may move it back to Editable state for further modification or
 choose to cancel the version.
 
-Cancelled state. A version is no longer relevant; no further edits may be
-introduced to the version.
+Cancelled state. A version is no longer relevant; it is no longer editable.
 
 ### Purchase Order Collaborative Sub-Workflow
 
 ![PO Collaborative Sub-Workflow](../images/0025-purchase-order-rfc/po_collaborative_sub_workflow.png)
 
-The Collaborative Sub-Workflow offers trade partners a means to manage
+The Collaborative Sub-Workflow enables trade partners a means to manage
```

Comment:
I think here it sounds kind of wonky with "a means" , if using "enables" it sounds better without "a means"